index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
```

Comment:
A small change here would be to set a meta element specifying the viewport. This is a good practice that ensures the code is up to standard with HTML best practices.